azconfig/client/src/main/java/com/azure/azconfig/implementation/package-info.java

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+/**
+ * This package contains classes for AzConfig Service SDK.
+ */
+package com.azure.azconfig.implementation;
```

Comment:
I want to be very clear in our package naming. To me, when I see an `implementation` package, it rings alarm bells - because I expect that this package contains no public API and it is for our internal use only. If this is not true, then the public API should be moved out of it. We should also ensure that when JavaDoc is generated that this package is excluded from generation (I believe it is in the parent pom, but just be certain).

In any case, I don't think the JavaDoc above is accurate?